src/loaders/api/api.js

```diff
@@ -154,6 +154,7 @@ export function setAPI (agentIdentifier, forceDrain) {
     if (typeof err === 'string') err = new Error(err)
     handle(SUPPORTABILITY_METRIC_CHANNEL, ['API/noticeError/called'], undefined, FEATURE_NAMES.metrics, instanceEE)
     handle('err', [err, now(), false, customAttributes], undefined, FEATURE_NAMES.jserrors, instanceEE)
+    handle('err', [err, now(), false, customAttributes], undefined, FEATURE_NAMES.sessionReplay, instanceEE)
```

Comment:
Just curious....why do we have to dispatch the error twice? Why cant the session replay just listing on the errors channel?

Reply:
The main (only?) reason would be that -- If jserrors feature drains before sessionReplay sets up its listeners, the data would be missed

Reply:
Hmm...could we track that all features have been loaded and instantiated before allowing the drain to commence? I think the best way to have cross-feature functionality is for featureA to listen for featureB events. Using the event emitter removes the direct dependency between the two features but it does complicate the event emitter a bit.

Reply:
yeah, i removed this code but this is a good call out.  I think we actually **do** already wait for all the features to be loaded before draining anyway, so this was likely redundant.